addons/website/static/src/snippets/s_process_steps/000.scss

```diff
@@ -1,9 +1,51 @@
+@mixin line($top, $right) {
```

Comment:
Maybe name the mixin by starting with the snippet name. All the scss files are regrouped and compiled together, `line` is too generic of a name and you might end up overriding something existing.

So something like `s-process-steps-line` (use `-` and not `_` in mixin names).